requirements.txt

```diff
@@ -3,3 +3,4 @@ mypy==0.931
 pytest
 pytest-timeout
 py-algorand-sdk
+types-dataclasses
```

Comment:
Note for reviewers:  Without installing `types-dataclasses`, Python 3.6 lacks `@dataclass` support.

Here's the error prior to installing the dependency:  https://github.com/algorand/pyteal/runs/5471035129?check_suite_focus=true.